src/pages/HomePage.tsx

```
@@ -11,10 +11,11 @@ import callToAction_Bg from "../assets/images/call-to-action-bg.jpg"
 import VidoeSection from "../components/VidoeSection";
 import img from "../assets/images/bitcoin-6992422_1920.jpg"
 import { useGetAllPackagesQuery } from "../slices/packageSlice";
+import LoadingComponent from "../components/LoadingComponent";
 
 const HomePage = () => {
 
-  const {data:packages} =useGetAllPackagesQuery({}) as any;
+  const {data:packages,isLoading} =useGetAllPackagesQuery({}) as any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)